src/Workspaces/Core/Portable/Workspace/Solution/Solution.cs

```diff
@@ -1202,11 +1202,14 @@ internal ImmutableArray<DocumentInfo> GetGeneratedDocuments(ProjectId projectId)
                 return ImmutableArray<DocumentInfo>.Empty;
             }
             var compilation = await this.CreateCompilationAsync(projectState, cancellationToken).ConfigureAwait(false);
+            ImmutableArray<Diagnostic> diagnostics;
```

Comment:
It looks like the value returned by compilation.GenerateSource into this local isn't checked. Perhaps it is worth adding a comment explaining why we don't need to do this here.